README.md

```diff
@@ -69,8 +69,21 @@ Considering the data provided above, the application should return the following
 ```
 
 ## Criteria
-You may use any language and framework, provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability; particularly with regards to architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.
+You may use any language and framework provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability, particularly regarding architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.
+
+Be aware that Origin will mainly take into consideration the following evaluation criteria:
+* How clean and organized your code is;
+* If you implemented the business rules correctly;
+* How good your automated tests are (qualitative over quantitative).
+
+Other important notes:
+* Develop a flexible recommendation engine, it will be essential to the take-home interview
```

Comment:
```suggestion
* Develop an extensible recommendation engine, it will be essential to the take-home interview
```

It's just a word, but I think the difference is important:
* Flexible is something that is aimed to be CHANGED
* Extensible is the Open-Closed Principle, something that we strongly care about during the Take Home Review. "How simple is to add a new rule or line of insurance? Does it take serious changes to the core implementation?"